scripts/social.js

```diff
@@ -14,10 +13,36 @@ social.prototype.toHtml = function() {
   return template(this);
 };
 
-socData.forEach(function(ele) {
-  soc.push(new social(ele));
-});
+social.all = [];
+
+social.fetchAll = function () {
+  if (localStorage.socialData) {
+    $.ajax({
+      type: 'HEAD',
+      url: 'data/socialData.json',
+      success: function(data,message,xhr) {
+        var eTag = xhr.getResponseHeader('eTag');
+        if (!localStorage.eTag || eTag !== localStorage.eTag) {
+          localStorage.eTag = eTag;
+        } else {
+          social.loadAll(JSON.parse(localStorage.socialData));
+        }
+      }
+    });
+    social.loadAll(JSON.parse(localStorage.socialData));
+    viewer.initIndexPage();
+  } else {
+    $.getJSON('data/socialData.json', function(data) {
+      var stringData = JSON.stringify(data);
+      localStorage.setItem('socialData',stringData);
+      social.loadAll(JSON.parse(localStorage.socialData));
+      viewer.initIndexPage();
```

Comment:
See the notes in your portfolio.js file for how to change this line.  You have the same situation here.